demos/tests/cases.py

```diff
@@ -413,6 +413,23 @@ def single_option_cases(key, *args):
                               '-dt': 'simple'}),
         ]
     )),
+
+    CppDemo(name='social_distance_demo', device_keys=['-d_det', '-d_reid'], test_cases=combine_cases(
+        TestCase(options={'-no_show': None,
+            **MONITORS,
+            '-i': DataDirectoryArg('person-detection-retail')}),
+        [
+            TestCase(options={'-m_det': ModelArg('person-detection-0200')}),
+            TestCase(options={'-m_det': ModelArg('person-detection-0201')}),
+            TestCase(options={'-m_det': ModelArg('person-detection-0202')}),
+            TestCase(options={'-m_det': ModelArg('person-detection-retail-0013')}),
+        ],
```

Comment:
Why not `single_option_cases()`

Reply:
I did that based on  [pedestrian tracker demo test cases](https://github.com/openvinotoolkit/open_model_zoo/blob/3fb7456dd4b94fe5f9930c0df480477655f9c456/demos/tests/cases.py#L310), this demo has similar set of networks.

Reply:
Should we use `single_option_cases()` in both demos' test cases?

Reply:
BTW, these demos are very similar, should we consider to combine them in the future?

Reply:
Yes, would be great